### ISSUE TITLE
incorrect term in exercise?

### ISSUE THREAD

**Issue by devbagdi** [hadley/r4ds#1524]
in exercise 4.2.5  - 3 did u mean shortest instead of fastest?

**Comment by mine-cetinkaya-rundel** [comment]
Nope, the goal is to calculate the speed as miles per hour, see https://mine-cetinkaya-rundel.github.io/r4ds-solutions/data-transform.html#exercises.
